### PR TITLE
Feature/alert component

### PR DESCRIPTION
**Pull request by hachiojidev**
image
image

### CODE REVIEW

File: src/components/Card/theme.ts [hunk 1]
```diff
@@ -9,15 +10,15 @@ export type CardTheme = {
 };
 
 export const light: CardTheme = {
-  background: "#FFFFFF",
+  background: lightColors.card,
```
Comment (hachiojidev) [inline src/components/Card/theme.ts:13]: This was being used many places so I put it in the theme.

File: src/theme/colors.ts [hunk 2]
```diff
@@ -26,11 +27,13 @@ export const lightColors: Colors = {
   textDisabled: "#BDC2C4",
   textSubtle: "#8f80ba",
   borderColor: "#E9EAEB",
+  card: "#FFFFFF",
 };
 
 export const darkColors: Colors = {
   ...baseColors,
   ...brandColors,
+  secondary: "#9A6AFF",
```
Comment (hachiojidev) [inline src/theme/colors.ts:36]: Turns out secondary is actually different on dark.

File: src/components/Svg/index.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ export { default as CommunityIcon } from "./Icons/Community";
 export { default as ErrorIcon } from "./Icons/Error";
```
Comment (RabbitDoge) [inline src/components/Svg/index.tsx:12]: Is there a reason why you don't export other icons ?

Comment (hachiojidev) [reply src/components/Svg/index.tsx:12]: Just missed it.

File: src/components/Alert/Alert.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    case variants.SUCCESS:
+      return CheckmarkCircleIcon;
+    case variants.INFO:
+    default:
+      return InfoIcon;
+  }
+};
+
+const Title = styled(Text)`
+  font-weight: 600;
```
Comment (RabbitDoge) [inline src/components/Alert/Alert.tsx:44]: you can just active the prop `bold` using .attrs, to keep consistency in case we change the bold style on Text

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**